samples/app-anonymous-users/nodejs/ClientApp/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Welcome to the app that support anonymous user support in teams meeting apps using create-react-app"
```

Comment:
Welcome to the app that support anonymous user support in Teams meeting apps including authentications.

Reply:
Fixed